updates/1.43.0.sh

```diff
@@ -35,8 +35,8 @@ install_app nc-restore
 [[ ! -f /.docker-image ]] && {
 
   # fix HPB with dynamic public IP
-  arch="$(uname -m)"
-  [[ "${arch}" =~ "armv7" ]] && arch="armv7"
+  arch="$(dpkg --print-architecture)"
+  [[ "${arch}" == "armhf" ]] || [[ "$(uname -m)" =~ "armv7" ]] && arch="armv7"
```

Comment:
```suggestion
  [[ "${arch}" =~ ^(armhf|arm)$ ]] && arch="armv7"
```

Reply:
Alright :) 

Yes I agree, I left the other one just in case, since it felt like you were sightly hesitating when we were talking in the chat about switching over to `dpkg` 🙏

I'll take a look at the suggestions as soon as I can after work 🥳